packages/public/inspection-capture-web/src/PhotoCaptureHUD/translations/de.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Put all translations in the `src/translations` directory instead of in the component directory itself : we should have only one translation file per language in the whole package.

For the format of the translation keys, there are two schools :
- Devs who prefer to have one translation key per word or phrase (ex "ok" or "message.hello") and re-use the same translation keys everytime the same message is used in the app
- Devs who prefer to have one translation key per translated text element on the screen, meaning that if you have 2 buttons saying "ok" you will have 2 different translation keys with the same value

In my case, I prefer the second option. Even though it means having a lot of different keys for the same words sometime, I think it's easier to maintain. Plus, if tomorrow the product team asks us to change the content of the button from "OK" to "YES", we just have to change the translation file instead of the react file.

Anyway, this means that in order to properly implement the second option, and if we want to have all the translations in on single file, the keys should be named something like that : `baseComponent.childComponent.otherChildComponent.element`

Of course, we don't have to put the entire names of each and every component, we can be a bit flexible and prefer readability. In your case, you could do something like :

```json
{
  "photo": {
    "hud": {
      "sight": {
        "addDamageBtn": "Schaden"
      },
      "addDamage": {
        "cancelBtn": "stornieren"
      }
    }
  }
}
`
``

I hope all of this is clear, if you have any question you can ask me on Slack

Reply:
Super clear I really appreciate your deep explanation ! Thanks